bats/tests/helpers/defaults.bash

```diff
@@ -74,6 +74,33 @@ if using_networking_tunnel && ! is_windows; then
     fatal "RD_USE_NETWORKING_TUNNEL only works on Windows"
 fi
 
+########################################################################
+: "${RD_MOUNT_TYPE:=reverse-sshfs}"
+
+validate_enum RD_MOUNT_TYPE reverse-sshfs 9p virtiofs
+
+if [ "$RD_MOUNT_TYPE" = "virtiofs" ] && ! using_vz_emulation; then
+    fatal "RD_MOUNT_TYPE=virtiofs only works with VZ emulation"
```

Comment:
Why to equal RD_MOUNT_TYPE to an error message? I'd suggest to just throw the message "virtiofs only works with VZ emulation"

Reply:
It tells the user that setting `RD_MOUNT_TYPE` to `virtiofs` only works when using VZ emulation. It is helpful because it tells the user where the setting comes from.